includes/tools.php

```diff
@@ -223,3 +226,145 @@ function verify_https_support() {
 	 */
 	return 200 === wp_remote_retrieve_response_code( $response );
 }
+
+/**
+ * Build a system environment report for submission along with any bug reports.
+ *
+ * This is largely based on (a simplified version) of Easy Digital Downloads' edd_tools_sysinfo_get()
+ * reporting function.
+ *
+ * @link https://github.com/easydigitaldownloads/easy-digital-downloads
```

Comment:
@pippinsplugins Thank you! Will note the link to this thread in the DocBlock for future reference.

![Phil Dunphy (of Modern Family) indicating that you're awesome](https://media.giphy.com/media/ClcWrARkrq1GM/giphy.gif)